includes/api/v1/class-wc-rest-coupons-controller.php

```diff
@@ -416,13 +419,13 @@ public function get_item_schema() {
 					'context'     => array( 'view', 'edit' ),
 				),
 				'date_created' => array(
-					'description' => __( "The date the coupon was created, in the site's timezone.", 'woocommerce' ),
+					'description' => __( "The date the coupon was created, as GMT.", 'woocommerce' ),
 					'type'        => 'date-time',
 					'context'     => array( 'view', 'edit' ),
 					'readonly'    => true,
 				),
 				'date_modified' => array(
-					'description' => __( "The date the coupon was last modified, in the site's timezone.", 'woocommerce' ),
+					'description' => __( "The date the coupon was last modified, as GMT.", 'woocommerce' ),
```

Comment:
On v1 needs to keep as `site's timezone` unless we already returned it as GMT before.